src/cpp/client/client_context.cc

```diff
@@ -71,13 +63,26 @@ void ClientContext::AddMetadata(const grpc::string& meta_key,
 
 void ClientContext::set_call(grpc_call* call,
                              const std::shared_ptr<ChannelInterface>& channel) {
+  {
```

Comment:
The existing gpr assert will cause a crash, and I thought the bug is asking
for something nicer.
On Aug 4, 2015 9:44 AM, "Craig Tiller" <notifications@github.com> wrote:

> In src/cpp/client/client_context.cc
> <https://github.com/grpc/grpc/pull/2775#discussion_r36211239>:
>
> > @@ -71,13 +63,26 @@ void ClientContext::AddMetadata(const grpc::string& meta_key,
> >
> >  void ClientContext::set_call(grpc_call* call,
> >                               const std::shared_ptr<ChannelInterface>& channel) {
> > +  {
>
> Do we want to do this or just crash?
>
> (I'd be ok with crashing)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grpc/grpc/pull/2775/files#r36211239>.
>